R/survival-survreg-tidiers.R

```diff
@@ -2,7 +2,8 @@
 #' @template title_desc_tidy
 #'
 #' @param x An `survreg` object returned from [survival::survreg()].
-#' @param conf.level confidence level for CI
+#' @template param_conflevel
```

Comment:
```suggestion
```

Both `conf.int` and `conf.level` are documented in the `param_confint` template (https://github.com/tidymodels/broom/blob/master/man-roxygen/param_confint.R#L3), so we can get away with just `#' @template param_confint` here.